source/our-stacks/android/index.md

```diff
@@ -1,6 +1,63 @@
 ---
 title: Android
-date: 2016-09-29 15:27:11
+date: 2016-10-06 17:37:40
 layout: stack
 ---
-Total Android
\ No newline at end of file
+
+Android developers at Nodes strive to build awesome apps using the Material Design Guidelines and Google provided design tools. 
+
+Custom designs are important to Nodes and helping client get the best possible Android experience are high priority for the Android devs.
```

Comment:
Maybe 'helping users get the best experience' sounds better than helping clients. Nevertheless, users/clients should be plural :)